social-network/src/App.css

```diff
@@ -0,0 +1,149 @@
+img { width: 100%; height: auto; }
+
+a { 
+  text-decoration: none; 
+  color: black;
+}
+
+#allArticles {
+  margin: 40px;
+}
+
+h2 {
+  font-size: 1.8em;
+  border-bottom: 1px solid #ccc;
+}
+
+h2 span {
+  border-bottom: 1px solid #666;
+}
+
+#yourArticles, #missedArticles {
+  display: grid;
+}
+
+#yourArticles {
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+  grid-column-gap: 1em;
+  grid-row-gap: 1em;
+}
+
+#missedArticles {
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+  grid-column-gap: 1em;
+  grid-row-gap: 1em;
+}
+
+.article {
+  border: 1px solid #ccc;
+}
+
+#yourArticles .article a {
+  display: grid;
```

Comment:
Nice use of CSS grid here! I'm a bit out of the loop when it comes to this, so it's great to see some examples used here and learn a little bit 👍